app/models/pawn.rb

```diff
@@ -1,2 +1,8 @@
 class Pawn < ChessPiece
+  def display
+    if color == "black"
+      return "&#9823;"
+    else "&#9817;"
+  end
```

Comment:
Should follow the same structure with a return "&#9817" being under the else statement.